101-dukes-eap-openshift/azuredeploy.json

```diff
@@ -0,0 +1,370 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "User name for the Virtual Machine and OpenShift Webconsole."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "User password for the OpenShift Webconsole"
+      }
+    },
+    "rhsmUser": {
+      "type": "string",
+      "metadata": {
+        "description": "Red Hat Subscription Manager User"
+      }
+    },
+    "rhsmPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Red Hat Subsciption Manager Password"
+      }
+    },
+    "rhsmPool": {
+      "type": "string",
+      "metadata": {
+        "description": "Red Hat Subscription Manager Pool(Must contain OpenShift entitlement)."
+      }
+    },
+    "sshKeyData": {
+      "type": "securestring",
+      "metadata": {
+        "description": "SSH RSA public key file as a string."
+      }
```

Comment:
add definitions for `dnsLabelPrefix` `eapUserName`, `eapPassword`, and `sshPassPhrase` here so they they are available for `azuredeploy.parameters.json`